src/ledger/LedgerTxn.h

```diff
@@ -458,6 +458,9 @@ class AbstractLedgerTxnParent
     // than a (real or stub) root LedgerTxn.
     virtual uint32_t prefetch(UnorderedSet<LedgerKey> const& keys) = 0;
 
+    // prepares to increase the capacity of pending changes up to "s" changes
+    virtual void reserveObjects(size_t s) = 0;
```

Comment:
I'm not 100% sure about this interface. `LedgerTxn` doesn't provide a convenient way to know how much space is already used, so probably it would be better to have `reserveAdditionalObjects` instead.

Reply:
I agree. In my branch I had a get/set but when I reduced it, I kept only the "set" which should be made relative